docs/tutorials/agentic_search/agentic_search_llm_generated_type.md

```diff
@@ -1,6 +1,6 @@
 ---
 layout: default
-title: Agentic search with a llmGenerated query type
+title: Getting Started with Agentic Search in OpenSearch
 parent: Agentic search
 nav_order: 30
 ---
```

Comment:
Can update the forum link with https://forum.opensearch.org/t/feedback-experimental-release-of-agentic-search-in-3-2/26470 on L12